Assets/Prefabs/UI/CreditsMenu.prefab

```diff
@@ -2423,9 +2423,13 @@ PrefabInstance:
     serializedVersion: 3
     m_TransformParent: {fileID: 3901135934142223114}
     m_Modifications:
+    - target: {fileID: 1685708173701239587, guid: 1b0c0786719c84648984b75e9055e89c, type: 3}
+      propertyPath: m_text
+      value: Livia Valenti
```

Comment:
Perhaps apply override to the perfab variant? Not too important unless she ends up contributing with other kinds of work somehow